### ISSUE TITLE
CASSSCF MPI parallelization error

### ISSUE THREAD

**Issue by VishalKJ** [nubakery/bagel#186]
Dear developers,

My CASSCF calculation crashes with "Max size reached in AugHess" error when i use more than one MPI process. However using the same input the CASSCF calculation completes smoothly when only using 1 MPI (1 or multiple OPENMP) process. Even when using the orbitals of a converged CASSCF calculation, the subsequent CASSCF crashes with 2 MPI processes. I even increased the value of "maxiter_micro" to 200 and "maxiter" to 200. 

Is the CASSCF module only supposed to be used serially ?

**Comment by shiozaki** [comment]
No, it works fine with MPI - it’s pretty well tested and used widely with
MPI. What is your config option and libraries version? Also what is the
bagel version? Did you test other molecules?

Sometimes your calculation only converges with the aid of numerical noise
(if you set up an ill defined calc), in which case this sort of things can
happen.

**Comment by shiozaki** [comment]
Try again without archive. Unfortunately boost archive is not MPI safe and
seems to break when the size is large. Also please provide the information
that I asked in the previous email.

**Comment by VishalKJ** [comment]
The bagel-version is recently downloaded from github in the first week of October 2019 compiled against gcc-8.3.1/OPENMPI 4.0.2.  It was configured as:

`../configure --prefix=/opt/share/sw/gcc-8.3.1/bagel_master201910_o3_avx2 --enable-dependency-tracking --enable-static=yes --enable-shared=yes --enable-mkl --with-mpi=openmpi --with-boost=/opt/share/libs/gcc-8.3.1/boost-1.71.0`

**Comment by VishalKJ** [comment]
the CXXFLAGS="-DNDEBUG -O3 -mavx2" was used

**Comment by shiozaki** [comment]
Sure, glad to hear it worked out for you. Toru
